libpq/plan.sh

```diff
@@ -0,0 +1,57 @@
+pkg_name=libpq
+pkg_origin=core
+pkg_version=9.6.6
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="LibPQ is the client side library for PostgreSQL, a powerful, open source object-relational database system."
+pkg_upstream_url="https://www.postgresql.org/"
+pkg_license=('PostgreSQL')
+pkg_source=https://ftp.postgresql.org/pub/source/v${pkg_version}/postgresql-${pkg_version}.tar.bz2
+pkg_dirname=postgresql-${pkg_version}
+pkg_filenam=postgresql-${pkg_version}.tar.gz
```

Comment:
The variable name has a typo. If it works like this, maybe there's no need to define it after all?

Reply:
Oh good catch.